etc/docker/Dockerfile

```diff
@@ -52,28 +52,38 @@ ENV OTBR_BUILD_DEPS \
   build-essential \
   psmisc
 
+# Required for OpenThread Backbone CI
+ENV OTBR_OT_BACKBONE_CI_DEPS curl ca-certificates
+
 # Required and installed during build (script/bootstrap) when RELEASE=1, could be removed
 ENV OTBR_NORELEASE_DEPS \
   cmake \
   cpputest \
   doxygen
 
-RUN apt-get update \
-  && apt-get install --no-install-recommends -y $OTBR_DOCKER_REQS $OTBR_DOCKER_DEPS \
-  && ln -fs /usr/share/zoneinfo/UTC /etc/localtime \
-  && ./script/bootstrap \
-  && ./script/setup \
-  && chmod 644 /etc/bind/named.conf.options \
-  && mv ./script /tmp \
+RUN apt-get update
```

Comment:
The reason why we collapsed everything into a single RUN command was to reduce the Docker images size (see #290). Is the Docker image size a concern with this modification?

Reply:
I built two Docker images:
- original: 1.37GB
- this PR: 1.68GB

Not sure if this sizes are reasonable.
Of course we can also have a separate Dockerfile for OT Backbone CI so that we don't change the original Dockerfile.

Reply:
@jwhui 
I have refactored the Dockerfile to use single RUN command. Please check.

Reply:
@simonlingoogle 
fyi, docker image size on commit 7a53c4c186672fd0a6810e82636f310225bba6bf (HEAD -> master, up/master)


```
➜  ot-br-posix git:(master) docker images
REPOSITORY                                       TAG                 IMAGE ID            CREATED             SIZE
openthread/otbr                                  latest              34c6fbf15241        15 seconds ago      552MB
```